front/src/applications/operationalStudies/hooks/usePathProjection.ts

```diff
@@ -78,17 +99,169 @@ const usePathProjection = (
+    if (!pathUsedForProjection) {
       return undefined;
     }
+
+    const operationalPointDistances: number[] = [];
+
+    // ===========================
+    // FAILED PATHFINDING HANDLING
+    // ===========================
+    // When pathfinding fails or path properties are unavailable, we still need to display
+    // operational points in the manchette and allow projection in the STD
+    // Strategy:
+    // 1. For each point in the path, try to match with infrastructure data
+    // 2. If a point is matched → use matched data with full extensions
+    // 3. If a point is not matched (e.g., NGE) → create a virtual point from the reference
+    if (pathfinding?.status !== 'success' || !pathProperties) {
+      const normalizedOps: PathProperties['operational_points'] = [];
+
+      requestedOperationalPoints.forEach((opRef, index) => {
+        const matchedOps = matchedOperationalPoints?.related_operational_points[index] || [];
+        const matchedOp = getStationFromOps(matchedOps);
+        const weight = 0; // Uniform weight for consistent manchette spacing
+        const position = index * FALLBACK_DISTANCE_MM; // Sequential positioning in manchette
+        if (index > 0) {
+          operationalPointDistances.push(FALLBACK_DISTANCE_MM); // Add distance for fallback positioning
+        }
+
+        if (matchedOp) {
+          // MATCHED: Point exists in infrastructure
+          normalizedOps.push({
+            id: matchedOp.id,
+            extensions: matchedOp.extensions,
+            part: matchedOp.parts.at(0) || { track: '', position: 0, extensions: undefined },
+            position,
+            weight,
+          });
+        } else {
+          // NOT MATCHED: Point doesn't exist in infrastructure (e.g., NGE point)
+          // Create virtual point from the reference
+
+          const virtualName =
+            ('trigram' in opRef && opRef.trigram) ||
+            ('uic' in opRef && opRef.uic?.toString()) ||
+            ('operational_point' in opRef && opRef.operational_point) ||
+            // TODO:  This logic is duplicated in multiple places.
+            // Extract this operational point name generation logic into a common utility function
+            (index === 0
+              ? t('main.requestedOrigin')
+              : index === requestedOperationalPoints.length - 1
+                ? t('main.requestedDestination')
+                : t('main.requestedPoint', { count: index + 1 }));
+
+          const virtualId = `virtual_op_${virtualName}`;
+
+          normalizedOps.push({
+            id: virtualId,
+            extensions: {
+              identifier: {
+                name: virtualName,
+                uic: ('uic' in opRef && opRef.uic) || 0,
+              },
+              sncf: {
+                ch: ('secondary_code' in opRef && opRef.secondary_code) || '',
+                ch_long_label: '',
+                ch_short_label: '',
+                ci: ('uic' in opRef && opRef.uic) || 0,
```

Comment:
`uic` is not the same thing than `ci`, you need to truncate the 2 first digits I think